sdk/eventhub/event-hubs/src/eventPosition.ts

```diff
@@ -76,113 +27,18 @@ export class EventPosition {
    * This information is only relevent if the event position was identified by an offset or sequence number.
    * Default value: `false`.
    */
-  isInclusive: boolean = false;
+  isInclusive?: boolean;
   /**
    * @property The enqueued time of the event identified by this position.
```

Comment:
```suggestion
   * @property The enqueued time in UTC of the event identified by this position.
```
Since we let the user specify `enqueuedOn` as a number, should we also mention that this is the [number of milliseconds since the Unix Epoch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime)?

Reply:
Wouldn't that be a given?

Reply:
Wouldn't that be a given?

Reply:
I'm not sure that it is. Unix time is in seconds, not milliseconds and since we're talking about a property that gets sent to a service (that could be written in any language) I think this is ambiguous. The only reason I knew it was milliseconds was because I looked at what our code did and say we were calling `getTime` on a date object. I couldn't actually find it documented in the service API docs (the amqp annotation just redirects to .NET docs that show you can specify a DateTime).

Reply:
fair point, any suggested wordsmithing? :)

Reply:
How about something like this?
> When provided as a number this value is the number of milliseconds since the Unix Epoch.